cypress/integration/oddname/oddname.js

```diff
@@ -44,6 +44,12 @@ function naughtyFileName(realName) {
 	)
 }
 
+let failsLeft = 5
+Cypress.on('fail', (error, runnable) => {
+	failsLeft--
+	throw error // throw error to have test still fail
+})
```

Comment:
So, I don't know about this one.
It seems like an odd thing to do, does cypress not support quick-fail on its own?

Reply:
It supports quick-fail in their "pro plan" (see https://github.com/cypress-io/cypress/issues/518#issuecomment-809514077).

Reply:
Maybe using https://github.com/javierbrea/cypress-fail-fast is an option, but that would add a dependency that is only used for one thing (and is implemented in 5 + 3 lines of code)

Reply:
> Maybe using [javierbrea/cypress-fail-fast](https://github.com/javierbrea/cypress-fail-fast) is an option, but that would add a dependency that is only used for one thing (and is implemented in 5 + 3 lines of code)

Let get this in then